### PR TITLE
chore: updated amplify.yml file

### PR DESCRIPTION
**Pull request by hdJerry**
# Setup app deployment for new project

## Acceptance

- [ ] AWS setup for new showcase with working url
- [ ] Adjust the production environment file to include the new deployment url

## Notes

- We use AWS Amplify for our deployments
- If you do not have access to the AWS account, please ask a manager for access
- You should have to modify the `amplify.yaml` in the top-level directory

### CODE REVIEW

File: amplify.yml [hunk 1]
```diff
@@ -125,6 +125,25 @@ applications:
...
+        paths:
+          - node_modules/**/*
+
+  - appRoot: solidjs-tailwind
+    frontend:
+      phases:
+        preBuild:
+          commands:
+            - nvm install --lts=gallium
+            - pnpm install
```
Comment (mbicknese) [inline amplify.yml:138]: Not sure if nvm supplies `pnpm` by default. Do we need to install it?

Comment (kodejuice) [reply amplify.yml:138]: I think we need to install it, no first class support yet

Comment (vyktoremario) [reply amplify.yml:138]: @hdJerry what's the update on this?

Comment (hdJerry) [reply amplify.yml:138]: We can use nvm then.

**Review by vyktoremario — APPROVED** [review]

**Pull request merged**